.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/
```

Comment:
Can you remove this from the repo? If you want to ignore IDE specific files, I recommend doing so with a [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).

Reply:
done